.github/workflows/test-coverage.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
```

Comment:
What about main?

Reply:
Just wanted to give it a try with only dev first as we're only pushing to main near a new version release, the idea is to add main next as well.